backend/src/nodes/image_util_nodes.py

```diff
@@ -396,12 +405,12 @@ def __init__(self):
                 ],
             ),
             DropDownInput(
```

Comment:
Could you extract this out into its own function? I would like to get rid of the copy paste.